src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -875,7 +855,7 @@ private String computeEnclosingLabel(FlowNode executorStepNode, List<FlowNode> h
         }
 
         @Override public String toString() {
-            return "ExecutorStepExecution.PlaceholderTask{runId=" + runId + ",label=" + label + ",context=" + context + ",cookie=" + cookie + ",auth=" + auth + '}';
+            return "ExecutorStepExecution.PlaceholderTask{label=" + label + ",context=" + context + '}';
```

Comment:
Reducing noise and redundancy. `CpsStepContext.toString` already encodes the `Run#externalizableId`; and `cookie` is now rarely interesting. `auth` was never interesting unless you installed `authorized-project`, now deprecated.